packages/react-components/src/localization/LocalizationProvider.tsx

```diff
@@ -183,8 +183,6 @@ export interface ComponentStrings {
   /** Strings for the participant list component */
   ParticipantList: ParticipantListStrings;
   /* @conditional-compile-remove(hide-attendee-name) */
-  /** Strings for the attendee role */
```

Comment:
Note for reviewers: this string was missing an interface causing the utils string component to mistype and have errors. This string is the only attendeerole string associated with it hence why it was mostly like typed as just "string" previously. This string should have been strongly typed and had it's own interface or be under another string interface.

To resolve this, we moved it out and added it to other interfaces where it is used. cc @JamesBurnside 

Anything related to AttendeeRole is associated with this bugfix.